backend/app/server/routes/post.py

```diff
@@ -9,18 +10,23 @@
     update_post,
     report_post
 )
+from ..controllers.upload import upload_image
 from ..models.post import (
     ResponseModel,
-    PostSchema,
```

Comment:
We don't need the PostSchema and UserSchema in models also, right?

Reply:
Nope

Reply:
But we can keep the whole schema as a reference in the `models/`

Reply:
Yup.. Cool 🎉